<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Words</class>
 <widget class="QWidget" name="Words">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Words</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>python_icon.png</normaloff>python_icon.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>words</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>language</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QComboBox" name="comboBox_select_language"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_select_operation"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>âœ“</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_message">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
